cw-orch-daemon/src/state.rs

```diff
@@ -150,6 +148,24 @@ impl DaemonState {
 
         serde_json::to_writer_pretty(File::create(&self.json_file_path).unwrap(), &json).unwrap();
     }
+
+    fn default_state_dir() -> Result<PathBuf, DaemonError> {
+        // The program panics if the home_dir is not set
+        dirs::home_dir()
+            .map(|home_dir| home_dir.join(".cw-orchestrator"))
+            .ok_or(DaemonError::StdErr(
+                format!(
+                    "Your machine doesn't have a home folder. Please specify the {} env variable to use cw-orchestrator", 
+                    CwOrchEnvVars::StateFolder.name()
```

Comment:
Any specific reason you decided to use `name()` method instead of implementing Display trait btw?